.github/scripts/create-vm.py

```diff
@@ -0,0 +1,462 @@
+import os
+import grpc
+import json
+import requests
+import math
+import logging
+import argparse
+import yaml
+import random
+import string
+from github import Github, Auth as GithubAuth
+from yandexcloud import SDK, RetryInterceptor, backoff_linear_with_jitter
+from yandex.cloud.compute.v1.instance_service_pb2_grpc import InstanceServiceStub
+from yandex.cloud.compute.v1.instance_pb2 import Instance
+from yandex.cloud.compute.v1.instance_service_pb2 import (
+    CreateInstanceRequest,
+    ResourcesSpec,
+    AttachedDiskSpec,
+    NetworkInterfaceSpec,
+    PrimaryAddressSpec,
+    OneToOneNatSpec,
+    DeleteInstanceRequest,
+    CreateInstanceMetadata,
+    DeleteInstanceMetadata,
+)
+from yandex.cloud.compute.v1.instance_pb2 import IpVersion
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s: %(levelname)s: %(message)s"
+)
+
+
+def output(key, value, secret=False):
```

Comment:
`append_output_file`? Also, upper-case if generally for constants in python, or for global variables, it is neither.

Reply:
well it is a global variable in that regard, env variable for the whole process